src/renderer/hooks/usePaneNodeSearchMenu.tsx

```diff
@@ -165,21 +174,58 @@ const groupSchemata = (
         schemata: cats.flatMap((c) => c.schemata).filter((n) => favorites.has(n.schemaId)),
     };
 
-    if (favs.schemata.length === 0) {
-        return cats;
-    }
+    const suggs: SuggestedSchemaGroup = {
+        type: 'suggested',
+        name: 'Suggested',
+        schemata: schemata.filter((n) => suggested.has(n.schemaId)),
+    };
 
-    return [favs, ...cats];
+    return [
+        ...(suggs.schemata.length ? [suggs] : []),
+        ...(favs.schemata.length ? [favs] : []),
+        ...cats,
+    ];
+};
+
+const renderGroupIcon = (categories: CategoryMap, group: SchemaGroup) => {
+    switch (group.type) {
+        case 'favorites':
+            return (
+                <StarIcon
+                    boxSize={3}
+                    color="yellow.500"
+                />
+            );
+        case 'suggested':
+            return (
+                <Icon
+                    as={VscLightbulbAutofix}
+                    boxSize={3}
+                    color="cyan.500"
+                />
+            );
+        case 'category':
+            return (
+                <IconFactory
+                    accentColor={getCategoryAccentColor(categories, group.categoryId)}
+                    boxSize={3}
+                    icon={group.category?.icon}
+                />
+            );
+        default:
+            assertNever(group);
```

Comment:
The exhaustive switch pattern needs a `return` here.

```suggestion
            return assertNever(group);
```